source/features/avoid-accidental-submissions.tsx

```diff
@@ -0,0 +1,30 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function onKeyDown(event: delegate.Event<KeyboardEvent, HTMLFormElement>): void {
+	if (event.key === 'Enter') {
+		event.preventDefault();
+		if (!pageDetect.isNewFile() && !pageDetect.isEditingFile()) {
+			select('textarea', event.delegateTarget.form)!.focus();
+		} else {
+			select('#commit-description-textarea', event.delegateTarget.form)!.focus();
+		}
+	}
+}
+
+function init(): void {
+	delegate(document, 'input#issue_title, input#pull_request_title, input#commit-summary-input', 'keydown', onKeyDown);
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isNewIssue,
+		() => pageDetect.isCompare() && select.exists('#new_pull_request'),
```

Comment:
I think
```suggestion
		pageDetect.isQuickPR,
```

Reply:
`isQuickPR` ([see](https://github.com/fregante/github-url-detection/blob/master/index.ts#L230)) matches 
`https://github.com/sindresorhus/refined-github/compare/master...branch-name?quick_pull=1`
but not also
`https://github.com/sindresorhus/refined-github/compare/master...branch-name`

Reply:
@yakov116 do you remember any tests that detect this? Or is it even needed? Any compare page could show that form I think, if there's any diff

Reply:
I don't bug I can check.

I think we had a feature that only ran on that page

Reply:
To facilitate you, it's only used in this feature: https://github.com/sindresorhus/refined-github/blob/main/source/features/linkify-branch-references.tsx

Reply:
I saw something similar in the very similar `prevent-duplicate-pr-submission`


Since it's just an event listener, lets' just use this

```suggestion
		pageDetect.isCompare(),
```

Reply:
You meant `pageDetect.isCompare`. Just changed 👍